node/runtime/pangolin/src/pallets/bridge_messages.rs

```diff
@@ -15,7 +15,7 @@ use pallet_fee_market::s2s::{
 frame_support::parameter_types! {
 	// Shared configurations.
 	pub const MaxMessagesToPruneAtOnce: MessageNonce = 8;
-	pub RootAccountForPayments: Option<AccountId> = Some(ConcatConverter::<_>::into_account_id((&b"root"[..]).into_h160()));
+	pub RootAccountForPayments: Option<AccountId> = Some(ConcatConverter::<_>::into_account_id((&b"root"[..]).derive_ethereum_address()));
```

Comment:
Why we need RootAccountForPayments here? Is it used for message users in smart contract to pay fees into? Need documents and we should not call it `RootAccounts`.

Rename to `PalletH160Addres`, and better to put it in pallet's associate type, and use Self to get it from 
```
type MessageDeliveryAndDispatchPayment =
		FeeMarketPayment<Self, WithPangolinFeeMarket, Ring, RootAccountForPayments>;
```